docs/setup/utilizing-grafana.md

```diff
@@ -1,2 +1,90 @@
 # Utilizing Grafana
 
+Grafana is a powerful open-source platform that allows users to visualize
+and monitor various metrics and data sources. In this guide, we will
+explore how to effectively use Grafana to monitor and analyze GitHub Actions.
+
+Grafana can be accessed through
+[the following URL](https://mon.scality.net/grafana/d/WidbLgPnk/).
+Once logged in, you will find two distinct boards: "GitHub Actions Costs"
+and "GitHub Actions Monitoring." These boards provide valuable insights
+into cost monitoring and general information about GitHub Actions.
```

Comment:
Those are specific to us, we also don't provide the dashboard in this particular project, if we want to mention it, then we should add the code, otherwise we can only mention that we can build grafana dashboards.

Reply:
Better?